app/src/renderer/scripts/axiosProxy.js

```diff
@@ -13,21 +13,15 @@ module.exports = (requestCounter = 0) => {
 
       const channel = `Axios/${requestCounter}`
 
-      console.log(`Request ${options.method.toUpperCase()} ${options.url}`)
```

Comment:
Duuuuude, I commented now 3 times on this line. ^^ Why do you want to remove it so bad?

Reply:
hahaha bc it's annoying since it spams the console and also bc I can't get the actual requests on the Network tab of the Inspector

Reply:
> I can't get the actual requests on the Network tab of the Inspector

yeah that is over for now since we need to proxy the request to the main thread :( will be back when we are on the web

> it's annoying since it spams the console

do you have an alternative? I would like to:
- see when a request is dispatched
- see when a request fails
- see when a request is successful

do we not need any of those?

Reply:
> yeah that is over for now since we need to proxy the request to the main thread :( will be back when we are on the web

Oh ok then I guess we'll have to leave them like that